frb_codegen/src/library/commands/ensure_tools_available.rs

```diff
@@ -8,7 +8,7 @@ use std::str::FromStr;
 
 lazy_static! {
     pub(crate) static ref FFIGEN_REQUIREMENT: VersionReq =
-        VersionReq::parse(">= 8.0.0, < 10.0.0").unwrap();
+        VersionReq::parse(">= 8.0.0").unwrap();
```

Comment:
nit: `< 12.0.0`

Reply:
What do you think about leaving this open? In the rarer case that a new `ffigen` version has an issue, user can always still fix it in their own `pubspec.yaml`. Or, the flutter_rust_bridge package itself in frb_dart can enforce this constraint. Most of the time new ffigen doesn't seem to cause a problem, so it's annoying to wait and synchronize with a new FRB version to allow the new ffigen when simply wanting to run `flutter pub upgrade --major-versions`

Reply:
Looks reasonable!

Reply:
Sounds good, I will leave it open here, and place the constraint in frb_dart/pubspec.yaml.